src/test/java/org/minions/devfund/yuri/linkedlist/InsertSortedTest.java

```diff
@@ -0,0 +1,177 @@
+package org.minions.devfund.yuri.linkedlist;
+
+import org.junit.Assert;
+import org.junit.Test;
+
+import java.util.LinkedList;
+
+/**
+ * // Source code recreated from a .class file by IntelliJ IDEA.
+ * // (powered by Fernflower decompiler)
+ */
+public class InsertSortedTest {
+    private static final int LISTFIRSTVALUE = 3;
+    private static final int LISTSECONDVALUE = 8;
+    private static final int LISTTHIRDVALUE = 11;
+
+    private static final int LISTSIZE1 = 4;
+    private static final int LISTSIZE2 = 3;
+
+    private static final int LISTNEWVALUE1 = 10;
+    private static final int LISTNEWVALUE2 = 14;
+    private static final int LISTNEWVALUE3 = 1;
+
+    private static final int LISTPOSITION3 = 3;
+    public static final String VALUE_ALREADY_IN_LIST = "to value already in list";
+    public static final String INSERT_SORTED_THROWS = "insertSorted throws ";
+    public static final String INSERT_SORTED_DOES_NOT_CORRECTLY_IN_FRONT_OF = "insertSorted does not correctly "
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Define a constant instead of duplicating this literal "insertSorted does not correctly " 3 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1192)